### PR TITLE
[Doc] Note about Eclipse IDE support

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

File: docs/user/manual.adoc [hunk 1]
```diff
@@ -331,6 +331,13 @@ If you get an error saying `No such file or directory: 'rust-analyzer'`, see the
 GNOME Builder 3.37.1 and newer has native `rust-analyzer` support.
 If the LSP binary is not available, GNOME Builder can install it when opening a Rust file.
 
+
+=== Eclipse IDE
+
+Prerequisites: You have installed the <<rust-analyzer-language-server-binary,`rust-analyzer` binary>>.
+
+Support for Rust development in the Eclipse IDE is provided by link:https://github.com/eclipse/corrosion[Eclipse Corrosion]. While it currently uses RLS as default, you can successfully configure it so the IDE will use `rust-analyzer` instead. To do so, with an Eclipse IDE where Corrosion is installed, just go to __Window > Preferences > Rust__ and edit the __Path to Rust Language Server__ entry to reference the path to `rust-analyzer`. You'll need to close and reopen all .rs and Cargo files, or to restart the IDE, for this change to take effect.
```
Comment (lnicola) [inline docs/user/manual.adoc:339]: We use a sentence-per-line style in the docs, it makes reading diffs easier.

Comment (mickaelistria) [reply docs/user/manual.adoc]: I changed that.

**Comment by mickaelistria** [comment]
Thanks!

**Pull request merged**